projects.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Projects
+parent: Get Involved
+order: 1
+---
+
+Please ask in #welcome:tomesh.net if you would like to learn more or contribute to any one of these projects.
+
+## Nodes/Network
+
+### Raspberry/Orange Pi Install Script + Bi-Weekly Virtual Mesh
```

Comment:
I don't know if we want recurring events listed as a project, not sure on that tho